docs/editor_agents/examples/index.md

```diff
@@ -30,6 +30,33 @@ def handle_object_hashes(
 
 An example use case of the above: Suppose that I have my own OCR model and I want to selectively run OCR on objects I've selected from the Encord app. You can then trigger your agent from the app and it'll appropriately send a list of objectHashes to your agent. Then via the dep_objects method above, it gives the agent immediate access to the object instance making it easier to integrate your OCR model.
 
+**Test the Agent**
+
+1. Save the above code as agent.py then in your current terminal, run the following command to run the agent in debug mode.
+
+    ```shell
+    functions-framework --target=handle_object_hashes --debug --source agent.py
+    ```
```

Comment:
```suggestion
1. Save the above code as `agent.py`. 
2. In your current terminal, run the following command to run the agent in debug mode.

```shell
functions-framework --target=handle_object_hashes --debug --source agent.py
```

Reply:
Think GH is bugging here. Missing three backticks in the end.